commit bf900d5f9b47322e414a4d07fbccfa33b8b0b3f3 (refs/changes/62/962/1)
Author: Angela Fong <angela.fong@joyent.com>
Date:   2016-11-28T17:26:01-08:00 (2 years, 10 months ago)
    
    HEAD-2332 Revert HEAD-2167 to include sdcsso logs for hermes upload

diff --git a/etc/logsets.json b/etc/logsets.json
index 7f92990..f019b96 100644
--- a/etc/logsets.json
+++ b/etc/logsets.json
@@ -230,6 +230,7 @@
       "redis",
       "sapi",
       "sdc",
+      "sdcsso",
       "ufds",
       "vmapi",
       "workflow"
